.eslintrc.json

```diff
@@ -14,7 +14,8 @@
   "globals": {
     "flowchart": true, // Blueprints support lib
     "__": true, // Zanata parsing cue
-    "N_": true // Zanata parsing cue
+    "N_": true, // Zanata parsing cue
+    "$window": true
```

Comment:
Ah, I cannot figure out how to show the old CC errors, was it the same message for both issues?

Hmm I don't see why it would say `$window` not defined in the `logout.state`, running it locally it looks like both ESLint issues were in `authorization.config.js`.